transports/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -12,56 +12,66 @@ related:
 
-When the number of messages per second exceeds this quota, the storage service responds with an [HTTP 503 Server Busy message](https://docs.microsoft.com/en-us/azure/media-services/media-services-encoding-error-codes). This message indicates that the platform is throttling the queue. If a single storage account is unable to handle an application's request rate, the application could leverage several different storage accounts using a storage account per endpoint. This ensures application scalability without saturating a single storage account. This also gives a discrete control over queue processing, based on the sensitivity and priority of the messages that are handled by different endpoints. For example, high priority endpoints could have more dedicated workers than low priority endpoints.
+For additional guidance on considerations when developing a system using Azure Storage Queues, see the article on [Performance and scalability checklist for Queue Storage](https://learn.microsoft.com/en-us/azure/storage/queues/storage-performance-checklist).
 
 > [!NOTE]
-> Using multiple storage accounts is currently NOT compatible with ServiceControl, it is necessary to use [ServiceControl transport adapter](/servicecontrol/transport-adapter.md) or multiple installations of ServiceControl for monitoring in such situation.
+> Use real Azure storage accounts. Do not use Azure storage emulator as it only supports a single fixed account named "devstoreaccount1".
 
+> [!NOTE]
+> There are limits to how much using multiple storage accounts increase throughput. Consider using [scale units as a comprehensive scaling strategy](https://learn.microsoft.com/en-us/azure/well-architected/performance-efficiency/scale-partition#choose-a-scaling-strategy) to address higher throughput and reliability needs.
 
-## Scaling Out
+## NServiceBus routing with multiple storage accounts
 
-A typical implementation uses a single storage account to send and receive messages. All endpoints are configured to receive and send messages using the same storage account.
+The preferred way to route when using multiple accounts is to register endpoints with their associated storage accounts. This should be done using connection string aliases   instead of raw connection strings.
 
-![Single storage account](azure01.png "width=500")
+### Connection string aliases
 
-When the number of instances of endpoints is increased, all endpoints continue reading and writing to the same storage account. Once the limit of 2,000 message/sec per queue or 20,000 message/sec per storage account is reached, Azure storage service throttles messages throughput.
+Using aliases for connection strings prevents exposing sensitive data that may be in connection strings. An empty string is the default alias which works when using a single account. When using multiple accounts, distinct aliases have to be registered for each storage account.
```

Comment:
Can this be simplified further?

Mentioning the empty string seems like we are leaking our internal implementation.

I tried to compress this a bit further.

```suggestion
Using distinct aliases for each storage account connection string prevents exposing sensitive data and is required when using multiple accounts.
```